### DISCUSSION TITLE
Collapse 组件缺少 size 类型信息， 在使用的时候， 会提示有错误

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#40876] [General]
Collapse 组件缺少 size 类型信息， 在使用的时候， 会提示有错误

**Comment by Yuiai01** [comment]
我看看~

**Comment by Yuiai01** [comment]
<img width="928" alt="image" src="https://user-images.githubusercontent.com/112228030/220575270-54590d0c-7a95-449a-b118-982019d68cf8.png">
我这有诶，能提供截图么

**Reply by ekuku1984** [reply]
我看看是不是我版本不对

**Reply by Yuiai01** [reply]
> 我看看是不是我版本不对

嗯嗯，Collapse 的 size 属性是在  5.2.0 版本发布的。
